testdata/configtest.json

```diff
@@ -2430,6 +2430,63 @@
      "supportedCurrencies": ""
     }
    ]
+  },
+  {
+   "name": "FTX",
+   "enabled": true,
+   "verbose": false,
+   "httpTimeout": 0,
+   "websocketResponseCheckTimeout": 0,
+   "websocketResponseMaxLimit": 0,
+   "websocketTrafficTimeout": 0,
+   "websocketOrderbookBufferLimit": 0,
+   "baseCurrencies": "USD",
+   "currencyPairs": {
+    "assetTypes": [
+     "spot",
+     "futures"
+    ],
+    "pairs": {
+        "futures": {
+         "enabled": "DOGE-PERP",
+         "available": "ADA-PERP,ADA-0626,ALGO-PERP,ALGO-0626,ALT-PERP,ALT-0626,ATOM-PERP,ATOM-0626,BCH-PERP,BCH-0626,BNB-PERP,BNB-0626,BSV-PERP,BSV-0626,BTC-PERP,BTC-MOVE,BTC-MOVE,BTC-MOVE,BTC-MOVE,BTC-MOVE,BTC-MOVE,BTC-0626,BTC-MOVE,BTC-0925,BTC-MOVE,BTC-MOVE,BTMX-PERP,BTMX-0626,DOGE-PERP,DOGE-0626,DRGN-PERP,DRGN-0626,EOS-PERP,EOS-0626,ETC-PERP,ETC-0626,ETH-PERP,ETH-0626,EXCH-PERP,EXCH-0626,HT-PERP,HT-0626,LEO-PERP,LEO-0626,LINK-PERP,LINK-0626,LTC-PERP,LTC-0626,MATIC-PERP,MATIC-0626,MID-PERP,MID-0626,OIL100-0525,OKB-PERP,OKB-0626,PAXG-PERP,PAXG-0626,BERNIE,BIDEN,BLOOMBERG,PETE,TRUMP,WARREN,PRIV-PERP,PRIV-0626,SHIT-PERP,SHIT-0626,TOMO-PERP,TOMO-0626,TRX-PERP,TRX-0626,TRYB-PERP,TRYB-0626,USDT-PERP,USDT-0626,XAUT-PERP,XAUT-0626,XRP-PERP,XRP-0626,XTZ-PERP,XTZ-0626",
+         "requestFormat": {
+          "uppercase": true,
+          "delimiter": "-"
+         },
+         "configFormat": {
+          "uppercase": true,
+          "delimiter": "-"
+         }
+        },
+        "spot": {
+         "enabled": "",
```

Comment:
re-enable spot pair

Reply:
Bump, can ./gocryptotrader --config=ttestdata/configtest.json to resolve this automatically or manually put in a pair `BTC/USDT`

Reply:
Bumperino 😆